modules/instrument_builder/jsx/react.questions.js

```diff
@@ -336,8 +336,8 @@ class DateOptions extends Component {
   // Render the HTML
   render() {
     // Truncate off the month and day from the date to only have the year.
-    let minYear = this.props.element.Options.MinDate.split('-')[0];
```

Comment:
both the comment above and the variable names are wrong.. why is this change being done? It seems unrelated to the title and I'm not sure of the impact of it on existing instruments.

Reply:
there's no impact on existing instruments i don't think. only within the instrument builder, the minyear and maxyear are being joined by 01-01 and 31-12 to make a full date which is unnecessary because linst only takes the year anyways. this line above that i'm removing then removes the month and day to only take the year. so i'm removing any addition/subtraction of day and month

Reply:
what about when loading existing instruments?

Reply:
hmmm something seems to be going wrong when you reload the linst and resave, i'll fix that now

Reply:
@driusan that is a separate bug reported here https://github.com/aces/Loris/issues/6669. this PR makes no changes to the linst file output

Reply:
> what about when loading existing instruments?

existing instruments only carry the year, so i don't understand why having the day and month within the instrument builder affects that